integration-tests/utils/awsClients.ts

```diff
@@ -0,0 +1,16 @@
+import { CloudWatchLogsClient } from '@aws-sdk/client-cloudwatch-logs'
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { SQSClient } from '@aws-sdk/client-sqs'
+
+const region = 'eu-west-2'
+const cloudWatchLogsClient = new CloudWatchLogsClient({
+  region: `${region}`
```

Comment:
You don't need to add string interpolation here. You can simply do `region: region`. Or `region: AWS_REGION` if you update to use the contant.